pkg/workflow/v1/validate.go

```diff
@@ -0,0 +1,9 @@
+package v1
+
+import "github.com/puppetlabs/nebula-sdk/pkg/util/typeutil"
+
+// Validates a yaml document according to the schema specification
+func Validate(y string) error {
```

Comment:
You may want to call this `ValidateYAML` as I expect we'll want to validate it in JSON form at some point too.